modules/access_level/main.tf

```diff
@@ -0,0 +1,31 @@
+resource "google_access_context_manager_access_level" "access-level" {
+  provider    = "google-beta"
+  parent      = "accessPolicies/${var.policy}"
+  name        = "accessPolicies/${var.policy}/accessLevels/${var.name}"
+  title       = "${var.name}"
+  description = "${var.description}"
+
+  basic = [
+    {
+      conditions = [{
+        ip_subnetworks         = ["${var.ip_subnetworks}"]
```

Comment:
`var.ip_subnetworks` is already a list. Is it supposed to be wrapped in an additional list?

Reply:
Fixed in https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/1/commits/0188f346668653adba9b655e436591eb6707135f